packages/epo-react-lib/package.json

```diff
@@ -54,7 +54,7 @@
     "@storybook/addon-interactions": "^6.5.12",
     "@storybook/addon-jest": "^6.5.12",
     "@storybook/addon-links": "^6.5.12",
-    "@storybook/builder-vite": "^0.2.2",
+    "@storybook/builder-vite": "^0.3.0",
```

Comment:
just updated the storybook stuff that touches Vite, when Storybook 7 comes out this plugin will be deprecated and I'll go through and update the others